src/main/java/world/bentobox/bentobox/util/heads/HeadCache.java

```diff
@@ -1,35 +1,153 @@
 package world.bentobox.bentobox.util.heads;
 
+import org.bukkit.Material;
 import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import java.lang.reflect.Field;
+import java.util.UUID;
+import com.mojang.authlib.GameProfile;
+import com.mojang.authlib.properties.Property;
+
+import world.bentobox.bentobox.BentoBox;
+
 
 /**
- * @since 1.14.0
- * @author tastybento
+ * This would allow to implement changeable player head for server owner.
+ * @since 1.14.1
+ * @author tastybento, BONNe1704
  */
-public class HeadCache {
-    private final ItemStack head;
+public class HeadCache
+{
+    // ---------------------------------------------------------------------
+    // Section: Variables
+    // ---------------------------------------------------------------------
+
+    /**
+     * Username for cached head.
+     */
+    private final String userName;
+
+    /**
+     * Userid for cached head.
+     */
+    private final UUID userId;
+
+    /**
+     * Encoded texture for given player head.
+     */
+    public final String encodedTexture;
```

Comment:
The Base64 link is not an image, it's the link to the particular skin on Mojang's Skins server. Yoi should rename this field to get rid of that confusion.